### PR TITLE
[#10580] Student view team page: Loading Indicator for loading teammates not centralised

### LINKED ISSUE #10580: Student view team page: Loading Indicator for loading teammates not centralised
- **Environment**: V7.0.2

**Steps to reproduce**
1. Go to student view team page and see the indicator be left aligned

### PR DESCRIPTION
**Pull request by jfrairigh**
Fixes #10580
Ready for review
**Outline of Solution**
 
In student-course-details-page.component.html I moved the directive that initiates the spinner image to wrap the section instead of being nested inside of it. 

s.ToReview

### CODE REVIEW

File: src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html [hunk 1]
```diff
@@ -85,12 +85,13 @@ <h5 class="d-inline"><strong> Sort By: </strong></h5>
   </div>
 </tm-loading-retry>
 
+<div *tmIsLoading="isLoadingTeammates">
 <div class="section">
```
Comment (t-cheepeng) [inline src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html:89]: indentation is not correct here?

Comment (jfrairigh) [reply src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html]: Changes have been made

**Comment by jfrairigh** [comment]
Changes have been made and successfully pushed to the PR. This is ready again for review.

**Review by t-cheepeng — APPROVED** [review]
LGTM

**Comment by jfrairigh** [comment]
Is there something else I need to do for this?

**Review by madanalogy — APPROVED** [review]
LGTM

**Pull request merged**